rx_signal_processing/rx_dsp_chain.cu

```diff
@@ -15,406 +15,234 @@
-uint32_t calculate_num_filter_taps(float rate, float transition_width) { // REVIEW #26 transition_width and transition_band are both used?
-    auto k = 3; //from formula 7-6 of Lyons text REVIEW #1 and #7 add some explanation for design of choice of k=3
-                // REPLY this feels well suited for our written documentation.
-    auto num_taps = k * (rate/transition_width); //REVIEW #5 provide units for rate & transition width
-                                                 //REPLY This can be added to the docstring
-
-    return num_taps; // REVIEW #33
-    //REVIEW #28 this function is returning a uint32; will this always round down? comment/document this
-}   //REPLY conversion from float to int never rounds, it always truncates. I'm unsure this needs to
-    //commented unless we need to choose and describe a particular choice of floor, ceil, or round
-
-std::vector<std::complex<float>> create_filter(uint32_t num_taps, float filter_cutoff,
-                                                float transition_width, // REVIEW #31 this line is over 80 characters, need to be consistent. #31
-                                                                        //REPLY I've been trying to stick to 100 lines. 80 is tough with all the templating
-                                    float rate) { // REVIEW #5 filter_cutoff, transition_width, rate
-                                                  //REPLY docstring
-    // REVIEW #3 explain algorithm including weight
-    // REPLY to be done in written documentation?
-    std::vector<double> desired_band_gain = {1.0,0.0};
-    std::vector<double> weight = {1.0,1.0};
-
-    auto filter_bands = create_normalized_lowpass_filter_bands(filter_cutoff,transition_width,
-                            rate);
-
-    /*remez returns number of taps + 1. Should we use num_taps + 1
-      or should we pass num_taps - 1 to remez?
-    */ //REVIEW #6 make this a TODO - does this depend on num_taps being odd or even?
-    //TODO(Keith): Investigate number of taps behaviour
-    std::vector<double> filter_taps(num_taps + 1,0.0);
-    auto converges = remez(filter_taps.data(), num_taps + 1, (filter_bands.size()/2), // REVIEW #32 declare and initialize number of bands for better readability
-                                                                                      //REPLY I am unsure what you mean as filterbands is both declared and initialized. Do you just not want the use of auto here?
-        filter_bands.data(),desired_band_gain.data(),weight.data(),BANDPASS,GRIDDENSITY); // REVIEW #15 are we passing the right values, should error check before passing these params
-                                                                                          //REPLY We should maybe error check the function inputs. we should have a strong error case in the failure to converge, or verify the filters saved to file.
-    if (converges < 0){ // REVIEW #10 remez returns True or False is this check correct?
```

Comment:
I see now that the comment was changed in remez file, I assume the comment was wrong?